lib/shared/bucketing-assembly-script/assembly/types/bucketedUserConfig.ts

```diff
@@ -129,7 +134,9 @@ export class SDKVariable extends JSON.Obj {
         public type: string,
         public key: string,
         public value: JSON.Value,
-        public evalReason: string | null
+        public _variation: string,
```

Comment:
there should be some accompanying changes somewhere that pass these new things into the constructor when building a bucketed config